src/pages/PostPage.tsx

```diff
@@ -1,8 +1,57 @@
-import { useParams } from 'react-router-dom';
+import { useCallback, useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { PostResponse } from '../types/response';
+import { getPost } from '../utils/post';
+
+type PostId = {
+  postId: string;
+};
```

Comment:
아래처럼 선언할 수 있을 것 같아요!
```suggestion
type PostId = string
```

Reply:
앗ㅎㅎ 너무 이상하네요 생각해보니 바로 수정가겠습니다!